### PR TITLE
Restructure to support ansible collections and fix sanity test issues.

### LINKED ISSUE #133: Directory structure for collection in Ansible Galaxy
The directory structure needs to be adjusted to meet the standards of the Ansible Collections, see
https://docs.ansible.com/ansible/latest/dev_guide/developing_collections.html#collection-structure

This includes creating a `galaxy.yml` file.

### LINKED ISSUE #134: Change README.rst to .md format
The Ansible Collections need a README.md.

Proposal is to convert the existing README.rst into .md format, and to remove README.rst.

### LINKED ISSUE #135: Remove email from author items
The items in the author fields in the METADATA blocks of the modules currently specify both Github nickname and email.

This should be changed to only specify the github nickname.

See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-fields

### LINKED ISSUE #137: Run Ansible sanity tests in CI
The Ansible Collections require the use of the Ansible sanity test suite (`ansible-test sanity`). See https://docs.ansible.com/ansible/latest/dev_guide/testing_sanity.html#testing-sanity.

This item is about addressing any issues that are reported by this test suite and about integrating its use into the CI system so that it is run regularly.

### PR DESCRIPTION
**Pull request by sabyadi**
See commit message.

Closes #133.
Closes #134.
Closes #135.
Closes #137.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 2668f6496cdd8f4c2d62d7129e59684d5954fc32
  * Invalid signoff. Commit message must end with  
    **Signed-off-by: .* <.*>**

Guidelines are available at https://github.com/zhmcclient/zhmc-ansible-modules/blob/master/CONTRIBUTING.rst

---
This message was auto-generated by https://gitcop.com

File: README.md [hunk 1]
```diff
@@ -0,0 +1,309 @@
+zhmc-ansible-modules - Ansible modules for the IBM Z HMC Web Services API
```
[the remark this answers is not in the document]
Comment (andy-maier) [reply README.md:16]: Seems this one is resolved.

File: docs/intro.rst [hunk 2]
```diff
@@ -255,7 +255,7 @@ All commands shown are to be executed in a bash shell on the control system.
 
    .. code-block:: bash
 
-      (myenv)$ pip install zhmc-ansible-modules
+      (myenv)# pip install zhmc-ansible-modules
```
Comment (andy-maier) [inline docs/intro.rst:258]: One does not need to be root to do this.
Usually, the `#` prompt indicates root, and the `$` prompt indicates non-root users.

**Review by andy-maier — CHANGES_REQUESTED** [review]
See comment.

**Comment by andy-maier** [comment]
I have resolved the merge conflict, and have resolved my own comment about the `#` vs. `$` prompt, in order to get this PR merged. We can still change the prompt again if there is disagreement.

File: docs/intro.rst (hunk 2 above)
Comment (andy-maier) [reply docs/intro.rst]: DONE

**Comment by andy-maier** [comment]
I backed out my change for '#' prompt back to '$', to allow it to succeed the checks.

**Pull request merged**